packages/workspace/src/core/hasher/hasher.ts

```diff
@@ -210,19 +213,16 @@ export class Hasher {
         ...implicitDepsFromPatterns,
 
         //TODO: vsavkin move the special cases into explicit ts support
-        'tsconfig.base.json',
         'package-lock.json',
         'yarn.lock',
         'pnpm-lock.yaml',
       ];
 
-      const fileHashes = [
-        ...fileNames.map((file) => {
-          const hash = this.fileHasher.hashFile(file);
-          return { file, hash };
-        }),
-        ...this.hashGlobalConfig(),
-      ];
+      const fileHashes = fileNames.map((file) => {
+        const hash = this.fileHasher.hashFile(file);
+        return { file, hash };
+      });
```

Comment:
Hmm that is a good question, I did this a while ago. I don't see a reason it should be so I will just add it back in.

Reply:
Oh actually, it was removed because of #1 the original PR description. 

> Removes the global nx.json hash, reasoning: the nx.json already has it's implicitDeps being hashed and the specific project in the nx.json projects object is also being hashed. This means we should not have to also hash the entire file.

Is that not true? If so I will leave the global hash in (as the PR stands)